tests/flow_typeapp_call/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,42 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`typeapp_call.js 1`] = `
+//@flow
+f<T>();
+f<T><U></U>;
+new C<T>;
+f<T>(e);
+o[e]<T>();
+f<T>(x)<U>(y);
+async <T>() => {};
+async <T>(): T => {}
+new C<T>(e);
+f<T>[e];
+new C<T>();
+o.m<T>();
+f<T>.0;
+o?.m<T>(e);
+o.m?.<T>(e);
+async<T>();
+f?.<T>(e);
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+//@flow
+f<T>();
+f < T > <U />;
```

Comment:
@existentialism Is this right?

Reply:
Yep, AFAIK, it makes sure it's JSX, here's associated parser test:
https://github.com/babel/babel/tree/master/packages/babel-parser/test/fixtures/flow/typeapp-call/rollback-jsx